### DISCUSSION TITLE
Integration not available

### DISCUSSION THREAD

**Discussion by Adriaanvr** [rainepretorius/olarm-ha-integration#102] [Q&A] [answered]
I am new to this, please bear with me. I can not find the Olarm integration on the HACS store. If I click on the link, I get the following message: _This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant._ My HA is up to date. Regards.

**Comment by rainepretorius** [comment, marked as answer]
You need to add i as a custom repository under hacs
